### PR TITLE
Add airflow one click app

### PR DESCRIPTION
**Pull request by Bass-03**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/airflow.yml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+            AIRFLOW_EMAIL: $$cap_airflow_email
+            AIRFLOW__SMTP__SMTP_HOST: $$cap_smtp_host
+            AIRFLOW__SMTP__SMTP_PORT: $$cap_smtp_port
+            AIRFLOW__SMTP__SMTP_STARTTLS: $$cap_smtp_starttls
+            AIRFLOW__SMTP__SMTP_SSL: $$cap_smtp_ssl
+            AIRFLOW__SMTP__SMTP_USER: $$cap_smtp_user
+            AIRFLOW__SMTP__SMTP_PASSWORD: $$cap_smtp_password
+            AIRFLOW__SMTP__SMTP_MAIL_FROM: $$cap_smtp_mail_from
+        ports:
+            - '8080:8080'
```
Comment (githubsaturn) [inline public/v4/apps/airflow.yml:166]: Why does it need a mapped port if it's already added as an HTTP container port?

**Review by githubsaturn — COMMENTED** [review]
Thanks for the PR! Just one minor comment.

File: public/v4/apps/airflow.yml (hunk 1 above)
Comment (Bass-03) [reply public/v4/apps/airflow.yml:166]: hey @githubsaturn 
That is the way the suggested docker compose file is set, I assumed I needed to define that port for that service, then tell caprover to map it to http and https ports.

My assumption is those two things are independent and both need to be explicitly defined.
Isn't that the case?

Comment (githubsaturn) [reply public/v4/apps/airflow.yml:166]: Yes, when you've added the port as the container port, the mapping is typically not needed anymore. I suggest removing it and do a quick test to make sure it works.

**Comment by Bass-03** [comment]
hey @githubsaturn 

You were right, it is not needed, I removed the ports and tested, all works as expected.
I checked and the image itself defaults to ports 8080 anyway, so all good.

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**